src/pages/docs/matchmaker/index.mdx

```diff
@@ -9,12 +9,20 @@ import { HeroPattern } from '@/components/HeroPattern';
 
 The Rivet Matchmaker is a casual matchmaker optimized to get your player connected to your game server in under a second in the optimal region & lobby.
 
-To integrate the Rivet matchmaker, your game needs to call these four endpoints:
-
-- **[matchmaker.lobbies.find](/docs/matchmaker/api/lobbies/find)**: Called on the game client to return the address to connect to & the player token
-- **[matchmaker.lobbies.ready](/docs/matchmaker/api/lobbies/ready)**: Called on the game server to notify the matchmaker that the game server is ready to accept players
-- **[matchmaker.players.connected](/docs/matchmaker/api/players/connected)**: Called on the game server to notify the matchmaker the player connected & verify the player token
-- **[matchmaker.players.disconnected](/docs/matchmaker/api/players/disconnected)**: Called on the game server to notify the matchmaker the player disconnected
+To integrate your servers to the Rivet Matchmaker system, you need to call these four endpoints in your server code.
+- **[matchmaker.lobbies.ready](/docs/matchmaker/api/lobbies/ready)**: Notifies the matchmaker that the server is ready to accept players
+- **[matchmaker.players.connected](/docs/matchmaker/api/players/connected)**: Notifies the matchmaker of a player connection while confirming the connection’s legitimacy
+- **[matchmaker.players.disconnected](/docs/matchmaker/api/players/disconnected)**: Notifies the matchmaker of a player disconnection
+- **[matchmaker.lobbies.setClosed](/docs/matchmaker/api/lobbies/set-closed)**: Notifies the matchmaker that the server is closing and no longer able to accept players
+
+Once integrated, you can call the following endpoints to find and join available lobbies in your client code.
```

Comment:
I would also mention lobbies.create.

Reply:
Would that be a clientside API?

Reply:
[need to add another comment on this to request changes]